api/Services/ImageService.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Microsoft.AspNetCore.Http;
+using System;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace iTechArt.Hotels.Api.Services
+{
+    public class ImageService
+    {
+        private string FolderName 
+        {
+            get { return Path.Combine("Resources", "Images"); }
+        }
+
+        private string PathToSave
+        {
+            get { return Path.Combine(Directory.GetCurrentDirectory(), FolderName); }
```

Comment:
Is there a better way than using GetCurrentDirectory?

Reply:
As an environment variable, will change this